src/structures/session.ts

```diff
@@ -769,7 +748,69 @@ export default abstract class Session {
                             `adelay=delays=${CLIP_PADDING_BEGINNING_MS}ms:all=1`,
                         ];
                     }
+                } else {
+                    if (encoderArgs["-af"]) {
+                        encoderArgs["-af"].push(
+                            `adelay=delays=${CLIP_PADDING_BEGINNING_MS}ms:all=1`,
+                        );
+                    } else {
+                        encoderArgs["-af"] = [
+                            `adelay=delays=${CLIP_PADDING_BEGINNING_MS}ms:all=1`,
+                        ];
+                    }
+
+                    for (let i = 0; i < 10; i++) {
```

Comment:
Can we avoid this duplications somehow?

`encoderArgs["-af"] = (encoderArgs["-af"] || []).push()?`